tests/test_visitors/test_ast/test_annotations/test_argument_annotations.py

```diff
@@ -227,6 +252,10 @@ def test_wrong_argument_annotation(
     correct_multiline_arguments,
     correct_arg_none_annotation,
```

Comment:
I guess that I have missed this point with nested `Literal`, `Union`, and `Annotated` types. Can you please also cover it here as well. I know that's not 100% related. But, it's pretty close.

Reply:
How I understand I need to cover `class attributes and variable annotations.`
Now, `visitor` checks only the functions. So I need some more time to do it.

Reply:
Sure, no hurry.